src/components/ArticleLanguage.js

```diff
@@ -47,7 +47,7 @@ export const ArticleLanguage = ({ lang, title, close }) => {
   }, [])
 
   return <div class='articlelanguage'>
-    <input type='text' placeholder={i18n('search-language-placeholder')} value={query} onInput={(e) => setQuery(e.target.value)} data-selectable />
+    <input type='text' placeholder={i18n('search-language-placeholder')} value={query} onInput={(e) => setQuery(e.target.value.trim())} data-selectable />
```

Comment:
Not related to your change: could you take this opportunity to remove the parenthesis around the `e` argument of the onInput callback? We don't have a strict standard but we generally omit it. Note that it also applies to the other file in this commit.